LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Project Jupyter Contributors
+Copyright (c) 2020 Project Jupyter Contributors
```

Comment:
Should we say Team KALE instead of Project Jupyter Contributers?

Reply:
I mentioned this in the hangouts message. We need all 4 group members and client's feedback.